template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/NuGet.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="aspnetcore-dev" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="msbuild" value="https://dotnet.myget.org/F/msbuild/api/v3/index.json" />
```

Comment:
The dotnet-core feed was present in all of the other 2.0 NuGet configs, packages outside the scope of the dependency graph of ASP.NET should still be available. We want to give access to the ecosystem, not just our ASP.NET deps at this point - if we still do that even with removing that feed, then this can happen.